tests/functional/conftest.py

```diff
@@ -63,17 +63,9 @@ def with_clean_db(db_engine):
 
     # We need to re-init the DB as it creates the default test group and
     # possibly more in future?
-    db.init(db_engine, authority=TEST_SETTINGS["h.authority"])
-
-
-@pytest.fixture(scope="session")
-def db_engine():
-    db_engine = db.make_engine(TEST_SETTINGS)
-    db.init(db_engine, authority=TEST_SETTINGS["h.authority"], should_create=True)
-
-    yield db_engine
-
-    db_engine.dispose()
```

Comment:
This is no longer needed now that the functests share the same `tests/conftest.py` DB fixtures with the unit tests.